Scripts/World/Chunk.cs

```diff
@@ -19,6 +19,8 @@ public class Chunk : Spatial
     SurfaceTool surfaceTool = new SurfaceTool();
     SpatialMaterial material = new SpatialMaterial();
 
+    System.Threading.Thread generationThread;
```

Comment:
access modifiers for every thread please?

Reply:
default private 😉

Reply:
still, would be nicer to see it, and `using System.Threading` would look nicer

Reply:
Godot defines a Thread class though 😡

Reply:
maybe there is a reason it does, have you read what Godot threads are?

Reply:
and at least do `using ST = System.Threading;` and then `ST.Thread`

Reply:
Ok sure, and yeah you can use either Godot threads or C# ones depending on what you need.
I assumed C# threads would be more stable, maybe with error messages other than '!E' is true.